api-objects/AppointmentSubmission.ts

```diff
@@ -0,0 +1,89 @@
+export class AppointmentSubmission {
+    constructor(clone?: AppointmentSubmission) {
+        if (clone) {
+            this.patient = clone.patient;
+            this.patientSurvey = clone.patientSurvey;
+        }
+    }
+
+    patientSurvey: PatientSurveyJSON;
+    patient: PatientJSON;
+
+    public exportToSQLVars() {
```

Comment:
this is one of the "shitty translation layers" that the guy who came in to our meeting, I forgot his name, was talking about.

Making it unnecessary might be more work than it's worth though.

Reply:
Lol, YUP. Luke _(That was Luke)_ and I both have had unpleasant experiences with connectors like these. My hope is that we will have some time during **week 9** to fix some of these connection breaks: The front end could benefit by using this class.